### PR TITLE
Update gradient corlor, font-size and margin in special offer section

### PR DESCRIPTION
**Pull request by CBinh710**
- Update padding of description in introduction section
- Update gradient to match with design
- Issue: [Link](https://github.com/users/CBinh710/projects/4/views/1?pane=issue&itemId=78700647)

### CODE REVIEW

File: Foodhut/src/styles/sections/special-offer.css [hunk 1]
```diff
@@ -17,11 +17,12 @@
 .special-offer-content {
   font-size: var(--font-size-normal);
   text-align: center;
-  width: 58%;
+  opacity: .8;
```
Comment (phuocduong-agilityio) [inline Foodhut/src/styles/sections/special-offer.css]: 0.8

Comment (CBinh710) [reply Foodhut/src/styles/sections/special-offer.css]: i fixed that

File: Foodhut/src/styles/variables.css [hunk 2]
```diff
@@ -7,7 +7,7 @@
   --color-neutral: #E8E8E8;
   --color-strong-neutral: #686D77;
   --color-warning: #FFB800;
-  --color-gradient-warm: linear-gradient(0deg, #fff 55.72%, #fdc55e 90%, #f54748 121%);  
+  --color-gradient-warm: linear-gradient(0deg, #fff 55.72%,#FCCCBE 90%, #FCBC88 121%);
```
Comment (lintran-agilityio) [inline Foodhut/src/styles/variables.css]: why should to uppercase color code?

Comment (CBinh710) [reply Foodhut/src/styles/variables.css]: It will match the format of the variables above.

Comment (phuocduong-agilityio) [reply Foodhut/src/styles/variables.css]: @CBinh710 Pls check your format

Comment (CBinh710) [reply Foodhut/src/styles/variables.css]: I fixed it

**Pull request merged**